tests/integration/pox/stacking/direct-stacking-with-bug.spec.ts

```diff
@@ -136,26 +129,35 @@ describe("testing solo stacker increase without bug", () => {
     expect(poxInfo.next_cycle.stacked_ustx).toBe(1_080_000_000_011_111);
 
     // Check Alice's table entry
-    const poxAddrInfo0 = await readRewardCyclePoxAddressList(network, 2, 0);
-    const poxAddrInfo0CV = hexToCV(poxAddrInfo0.data) as SomeCV<TupleCV>;
-    expect((poxAddrInfo0CV.value.data["total-ustx"] as UIntCV).value).toBe(
+    const poxAddrInfo0 = (await readRewardCyclePoxAddressForAddress(
+      network,
+      2,
+      Accounts.FAUCET.stxAddress
+    )) as Record<string, ClarityValue>;
+    expect((poxAddrInfo0["total-ustx"] as UIntCV).value).toBe(
```

Comment:
can be the same as above:
`expect((poxAddrInfo0["total-ustx"] as UIntCV).value).toEqual(uintCV(900_000_000_000_001))`